lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -30,13 +22,9 @@ def initialize(engine_config:)
         end
 
         def run(file)
-          if (skip_reason = skip?(file))
-            $stderr.puts("Skipping file #{file} because #{skip_reason}")
-          else
-            process_file(file)
-          end
+          build_sexp(node(file), file)
         rescue => ex
-          if RESCUABLE_ERRORS.map { |klass| ex.instance_of?(klass) }.include?(true)
+          if ex.is_a?(CC::Parser::Client::HTTPError)
             $stderr.puts("Skipping file #{file} due to exception (#{ex.class}): #{ex.message}\n#{ex.backtrace.join("\n")}")
```

Comment:
maybe we should update this to better match the error handling in structure? https://github.com/codeclimate/codeclimate-structure/commit/02b008264f04472ac38b50742405264ce41629b1